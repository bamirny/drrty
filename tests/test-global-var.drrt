int x;
int y;



def int main() {
	int z;

	x=12;
	y=15;
	z = x+y;
	return z;
}	