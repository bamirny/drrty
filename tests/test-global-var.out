FAILED
  ./test-global-var.exe failed on ./test-global-var.exe
